utils.go

```diff
@@ -456,3 +457,18 @@ func TimeElapsed(now, then time.Time, abbreviationFormat string, fullTime bool)
 	}
 	return parts[0] + text
 }
+
+func voteVersion(params *chaincfg.Params) uint32 {
```

Comment:
Please, add a comment here that this was borrowed from upstream and should be updated whenever the upstream version is also updated. The comment should also include a link to the LOC upstream where this function may be found.